src/main/java/uk/gov/companieshouse/web/companydocuments/util/YamlResourceMapper.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.companieshouse.web.companydocuments.util;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.util.Map;
+import org.springframework.stereotype.Component;
+import org.yaml.snakeyaml.Yaml;
+
+@Component
+public class YamlResourceMapper {
+
+    public Map<String, Object> fetchYamlFromFile(String filePath) {
+
+        try {
+            return new Yaml().load(
+                    new FileInputStream(new File(filePath)));
+
+        } catch (FileNotFoundException e) {
+
+            throw new RuntimeException(e);
```

Comment:
A message would be handy. I've looked at the Java source and not every throw of FileNotFound includes a message, so something along the lines of ("Could not load YAML file: " + filePath, e).
Also throwing RuntimeException makes Sonar wince - perhaps IllegalArgumentEx, or IllegalStateEx?

Reply:
Addressed in [af9255a7](https://github.com/companieshouse/company-documents.web.ch.gov.uk/pull/10/commits/042c7af9255a7fc7fc142371427169525f946764)